src/ethstorage.js

```diff
@@ -21,9 +21,9 @@ export class EthStorage {
     #blobUploader;
 
     static async create(config) {
-        const {rpc} = config;
+        const {rpc, contractAddr} = config;
```

Comment:
The spec and readme should also be updated?

Reply:
https://github.com/ethstorage/ethstorage-sdk/blob/main/spec.md#create , This should also need to be updated.
such as: `address (string, optional): `